pkg/monitors/cpu/cpu.go

```diff
@@ -240,8 +240,7 @@ func cpuTimeStatTototalUsed(t *cpu.TimesStat) *totalUsed {
 		t.Softirq +
 		t.Steal +
 		t.Guest +
-		t.GuestNice +
-		t.Stolen
+		t.GuestNice
```

Comment:
What does this have to do with "Improve performance of host observer"?

Reply:
Also, we probably need the stolen metric for consistency.

Reply:
See https://github.com/signalfx/signalfx-agent/pull/1093/files#diff-1d0f8cf86bd7c90de47e9a231a73e489R234.